src/Microsoft.PowerShell.Commands.Management/commands/management/TestConnectionCommand.cs

```diff
@@ -133,19 +134,22 @@ public class TestConnectionCommand : PSCmdlet, IDisposable
         public int MaxHops { get; set; } = DefaultMaxHops;
 
         /// <summary>
-        /// Gets or sets the number of ping attempts.
-        /// The default (from Windows) is 4 times.
+        /// Gets or sets the number of test attempts.
+        /// The value is -1 if not set explicitly as a parameter
+        /// If not set explicitly, the value is set depending on type of test (ping = 4, tcp = 1)
```

Comment:
And then just remember to update the documentation comments. 😊

Reply:
Updated! 😁